lib/mixlib/install/product.rb

```diff
@@ -198,18 +185,6 @@ def lookup(key, version = :latest)
     config_file "/etc/opscode/chef-server.rb"
   end
 
-  product "chef-server-ha-provisioning" do
```

Comment:
I didn't see any builds of this product in either current or stable:
https://bintray.com/chef/current
https://bintray.com/chef/stable

It could be it hasn't been built in a while and has been reaped from current. Can you provide some context @irvingpop?